pkg/kubectl/cmd/describe.go

```diff
@@ -120,11 +120,14 @@ func RunDescribe(f cmdutil.Factory, out, cmdErr io.Writer, cmd *cobra.Command, a
 		return err
 	}
 
+	// include the uninitialized objects by default
+	includeUninitialized := true
 	r := builder.
 		ContinueOnError().
 		NamespaceParam(cmdNamespace).DefaultNamespace().AllNamespaces(allNamespaces).
 		FilenameParam(enforceNamespace, options).
 		SelectorParam(selector).
+		IncludeUninitializedParam(includeUninitialized).
```

Comment:
@ahmetb As @caesarxuchao commented in [#49035](https://github.com/kubernetes/kubernetes/issues/49035#issuecomment-322348893)

> Both should includes uninitialized objects. --include-uninitialized doesn't affect them at all.

So I keep it unchanged.